tests/unit_tests/vc/vc_array_grid_builder.cpp

```diff
@@ -0,0 +1,9 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
This file is not added to the `CMakeLists.txt`

Reply:
Excellent catch